CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+## Welcome!
+
+Hello! It's awesome that you want to contribute to the Discord bridge, by doing
+so you are helping our users and the wider Matrix ecosystem. This document
+lists the requirements for your work to be accepted into our repository.
+
+For clarity reasons, work can either be a pull request or an issue. We find
+both quite valuable to the project. One last thing before we begin is that you
+must abide by our licence, which is listed in LICENCE and is the Apache 2
+licence.
+
+Most importantly we are a welcoming project and will be happy to review any
+works, no matter the skill level of the submitter. Everyone must start
+somewhere and if you feel you are somewhat confident in being able to help then
+we are doing our jobs properly!
+
+## Quickfire tips (or TL;DR)
+
+* Always work off the **develop** branch. We won't accept merges into any other branch except from special occasions.
+* We follow the [Matrix Code of Conduct](https://matrix.org/docs/guides/code_of_conduct.html) and will not accept or entertain works from individuals who break it. We believe in considerate and respectful members above accepting works, regardless of quality.
+  * This includes possible bans from any room(s) involved with the project.
+* We are limited to accepting work over Github. Please use its interface when submitting work items.
+* Discussion of ideas for the bridge and work items should be in [#discord:half-shot.uk](https://matrix.to/#/#discord:half-shot.uk).
```

Comment:
Add this? ![](https://img.shields.io/matrix/discord:half-shot.uk.svg?server_fqdn=matrix.half-shot.uk) `![](https://img.shields.io/matrix/discord:half-shot.uk.svg?server_fqdn=matrix.half-shot.uk)`

Reply:
Maybe not a good idea, I think the icon would be a little jarring. it's also on the README